app/uk/gov/hmrc/tariffclassificationfrontend/views/ruling_details.scala.html

```diff
@@ -70,24 +70,24 @@ <h4 class="heading-medium mt-0">Ruling</h4>
 
                     <hr/>
 
-                    @defining(c.decision.get) { d =>
+                    @defining(c.decision.get) { d: Decision =>
 
                         <div class="grid-row">
                             <div id="ruling_bindingCommodityCode" class="column-one-third">
                                 <p>Binding Commodity Code</p>
                             </div>
                             <div class="column-two-thirds">
-                                <p id="ruling_bindingCommodityCodeValue">@{d.bindingCommodityCode}</p>
+                                <p id="ruling_bindingCommodityCodeValue">@d.bindingCommodityCode</p>
                             </div>
                         </div>
 
                         @if(d.methodSearch.isDefined) {
                             <div class="grid-row">
                                 <div id="ruling_searches" class="column-one-third">
-                                    <p>Searches</p>
+                                    <p>Searches Performed</p>
                                 </div>
                                 <div class="column-two-thirds">
-                                    <p id="ruling_searchesValue">@{d.methodSearch.get}</p>
+                                    <p id="ruling_searchesValue">@d.methodSearch</p>
```

Comment:
`methodSearch` is an `Option[String]`
We do not want to print the "Some" wrapper.

Do you agree?

Reply:
I am going to make this change for you :)